pkg/reconciler/pubsub/pullsubscription/k8s/controller.go

```diff
@@ -14,28 +14,30 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package pullsubscription
+// TODO better package name?
+package k8s
```

Comment:
fixed?  static?  singleton?  It seems like a name like that may be somewhat more self-describing for what you care about (ie, the scaling strategy).  Don't care a lot either way (though a less conflicting package name would be nice)

Reply:
Agreed, changed it to static. Seems better. Thanks for the suggestion.